packages/opencensus-core/src/trace/model/tracer.ts

```diff
@@ -52,7 +52,6 @@ export class Tracer implements types.Tracer {
   constructor() {
     this.activeLocal = false;
     this.contextManager = cls.createNamespace();
-    this.clearCurrentTrace();
```

Comment:
What is the purpose of this deletion?

Reply:
Some unit tests were failing. Both cases related to context issues.

Reply:
`clearCurrentTrace` is part of the public API but it will throw when using `continuation-local-storage`. Could you add a TODO to address this in a follow-up PR?